Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -735,6 +735,24 @@ Surface of the output mesh generated with a very small `facet_distance`
 without the weights (left, 25563 vertices) and with the weights (right, 19936 vertices).
 \cgalFigureEnd
 
+\subsubsection Mesh_3DomainsFrom3DImagesWithFeatures Domains From Segmented 3D Images, with Feature Detection
```

Comment:
This is in [4.4 Domains From 3D Images](https://doc.cgal.org/latest/Mesh_3/index.html#title26), but 4.6 is "Meshing Domains with Sharp Features" (see also https://github.com/CGAL/cgal/pull/6129/files?diff=split&w=0#r1023034964)